README.md

```diff
@@ -5,18 +5,21 @@ The Ballerina standard library
 
 |Level| Modules | Latest Version | Build | Last Commit | Open Issues | Open Pull Requests |
 |:---:|:---:|:---:|:---:|:---:|:---:|:---:|
+|0|[xmlutils](https://github.com/ballerina-platform/module-ballerina-xmlutils)|[![GitHub Release](https://img.shields.io/github/release/ballerina-platform/module-ballerina-xmlutils.svg?label=)](https://github.com/ballerina-platform/module-ballerina-xmlutils/releases)| [![Build](https://github.com/ballerina-platform/module-ballerina-xmlutils/workflows/Build/badge.svg)](https://github.com/ballerina-platform/module-ballerina-xmlutils/actions?query=workflow%3ABuild)|[![GitHub Last Commit](https://img.shields.io/github/last-commit/ballerina-platform/module-ballerina-xmlutils.svg?label=)](https://github.com/ballerina-platform/module-ballerina-xmlutils/commits/master)|[![Github issues](https://img.shields.io/github/issues/ballerina-platform/ballerina-standard-library/module/xmlutils.svg?label=)](https://github.com/ballerina-platform/ballerina-standard-library/labels/module%2Fxmlutils) | [![GitHub pull-requests](https://img.shields.io/github/issues-pr/ballerina-platform/module-ballerina-xmlutils.svg?label=)](https://github.com/ballerina-platform/module-ballerina-xmlutils/pulls)|
```

Comment:
Let's start from Level 1. No point of level 0, isn't it?

Reply:
Do we need to move this module to level 1 or no need to update zero level module here?